kernelci/config/storage.py

```diff
@@ -11,6 +11,8 @@
 class Storage(YAMLConfigObject):
     """Base configuration class for Storage implementations"""
 
+    yaml_tag = '!Storage'
```

Comment:
Where is `yaml_tag` being used?

Reply:
Every time a base class `Storage` config object get deserialised from YAML.  All the current storage types defined in the file have inherited classes but it's entirely possible to have a storage type that just relies on the base Storage config.

Reply:
See also: https://pyyaml.org/wiki/PyYAMLDocumentation